db/schema.rb

```diff
@@ -1082,7 +1082,7 @@
     t.string "oidc_issuer_endpoint"
     t.bigint "lock_version", default: 0, null: false
     t.string "authentication_method"
-    t.text "policies_config"
+    t.text "policies_config", limit: 16777215
```

Comment:
hmm, have you actually tested this? I think the migration version should also be updated in schema. Also the schemas for the other databases need also have the latest schema version despite no structure changes are done.

Reply:
It's correct, this migration originally updated the migration version for mysql, postgres and oracle schemas. Later I merged master into this branch, including a newer migration with a higher version number. For the new schema, this PR migration is assumed to be applied, so only the actual changes need to be present in the schema, not the version number. For postgres and oracle schemas, since there's no actual DDL updates, no changes are needed in the schema.

Reply:
it wou;d be an issue for anybody that clones the code because rails will not detect a missing migration. Or will it?

Reply:
I think it will because the version number wouldn't be inserted into the `schema_migrations` table in that case (AFAIK).

I can regenerate the migration to take another higher version number, but if I do, it'll be just before merging to ensure I don't need to repeat the process.